BasicSample/Assets/Build/BuildApp.cs

```diff
@@ -68,5 +77,33 @@ private static string[] SplitSceneList(string sceneList)
             return (from scene in sceneList.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
                     select scene.Trim()).ToArray();
         }
+
+        private static void SetUpAppRemoting(BuildTargetGroup targetGroup)
+        {
+            Debug.Log($"Setting up app remoting for {targetGroup}");
+
+            const string AppRemotingPlugin = "Microsoft.MixedReality.OpenXR.Remoting.AppRemotingPlugin";
+            Type appRemotingFeature = typeof(AppRemoting).Assembly.GetType(AppRemotingPlugin);
+            if (appRemotingFeature == null)
+            {
+                Debug.LogError($"Could not find {AppRemotingPlugin}. Has this class been removed or renamed?");
+                return;
+            }
+
+            FeatureHelpers.RefreshFeatures(targetGroup);
+            OpenXRFeature feature = OpenXRSettings.ActiveBuildTargetInstance.GetFeature(appRemotingFeature);
+            if (feature == null)
+            {
+                Debug.LogError($"Could not load {AppRemotingPlugin} as an OpenXR feature. Has this class been removed or renamed?");
```

Comment:
nit:  should this error message use "appRemotingFeature.Name" or short name ?

Reply:
Unfortunately, since the app remoting feature type is `internal` I can only get it via this string. I think that's okay for now, but do agree that a more friendly name would be nicer at some point.